lib/run/index.js

```diff
@@ -136,13 +136,19 @@ module.exports = function (options, callback) {
         }
 
         // sets entrypoint to execute if options.folder is specified.
-        if (options.folder) {
+        if (options.folder && !options.itemFromFolder) {
```

Comment:
This makes the two options mutually exclusive. Let users provide a folder and requests too. 

If both provided then the supplied request command will add the folder and the selected requests.